pokemongo_bot/event_handlers/telegram_handler.py

```diff
@@ -96,19 +96,11 @@ def run(self):
                         outMessage = "Telegram bot setup not yet complete (master = null). Please enter your userid {} into bot configuration file.".format(update.message.from_user.id)
                         self.bot.logger.warn(outMessage)
                         continue
-                    if self.master not in [update.message.from_user.id, "@{}".format(update.message.from_user.username)]:
```

Comment:
@solderzzc I agree, but I'd be curious to know how many people actually use the username in their config, since that doesn't actually work properly.